pkg/restore/util/util.go

```diff
@@ -0,0 +1,12 @@
+package util
```

Comment:
It's better to move this file into `pkg/util/` directory.

Reply:
+1